packages/graphback-runtime-mongodb/src/MongoDBDataProvider.ts

```diff
@@ -102,7 +102,19 @@ export class MongoDBDataProvider<Type = any, GraphbackContext = any> implements
   }
 
   public async findBy(filter: AdvancedFilter, orderBy?: GraphbackOrderBy, page?: GraphbackPage): Promise<Type[]> {
-    const query = this.db.collection(this.collectionName).find(buildQuery(filter));
+    const sortOrder: { [fieldName: string]: 1 | -1 } = {};
+    if (orderBy) {
+      if (orderBy.field) {
+        sortOrder[orderBy.field] = 1;
+      }
+      if (orderBy.order) {
+        if (orderBy.order.toLowerCase() === 'desc') {
+          sortOrder[orderBy.field] = -1;
+        }
+      }
+      console.log('sortorder: ', JSON.stringify(sortOrder));
```

Comment:
We need better logging on both knex and mongo. It will be cool to see queries without debugging. separate to this PR

Reply:
For the moment I will remove it